recipe/patches/0001-windows-ssl-lib-names.patch

```diff
@@ -11,23 +11,21 @@ Co-Authored-By: Marius van Niekerk <marius.v.niekerk@gmail.com>
  1 file changed, 4 insertions(+), 4 deletions(-)
 
 diff --git a/setup.py b/setup.py
-index ec27fe478f..bea7f92e15 100644
+index 94aa7793a7..5892a3288f 100644
 --- a/setup.py
 +++ b/setup.py
-@@ -329,10 +329,10 @@ if "win32" in sys.platform:
-         'ws2_32',
+@@ -363,10 +363,10 @@ if "win32" in sys.platform:
+         "ws2_32",
      )
  if BUILD_WITH_SYSTEM_OPENSSL:
 -    EXTENSION_LIBRARIES += (
--        'ssl',
--        'crypto',
+-        "ssl",
+-        "crypto",
 -    )
 +    if not "win32" in sys.platform:
```

Comment:
Some parts of these patches seem to support win32, which we don't do any more. We could probably remove this stuff

Reply:
win32 is windows 64 bits in this case, see https://docs.python.org/3/library/sys.html#sys.platform.